### ISSUE TITLE
pip install fails

### ISSUE THREAD

**Issue by carlodef** [mapbox/landsat8-qa#8]
On macOS:

```
$ pip install rio-l8qa
Collecting rio-l8qa
  Using cached rio-l8qa-0.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/mf/5rkwpjnj1vj502f9_y2lcpjw0000gn/T/pip-build-hwjit2dk/rio-l8qa/setup.py", line 15, in <module>
        with codecs_open('README.md', encoding='utf-8') as f:
      File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/codecs.py", line 895, in open
        file = builtins.open(filename, mode, buffering)
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/mf/5rkwpjnj1vj502f9_y2lcpjw0000gn/T/pip-build-hwjit2dk/rio-l8qa/
```

**Comment by perrygeo** [comment]
@carlodef Thanks for catching this! I just released `0.1.1` which should work. Let me know how it goes.

**Comment by carlodef** [comment]
@perrygeo The new version works. Thank you!
